tools/test-case-generators/repos.json

```diff
@@ -225,7 +225,8 @@
         "image_type_tags": [
-          "azure-rhui"
+          "azure-rhui",
+          "azure-eap"
```

Comment:
I wonder whether it would make sense to move changes done to `tools/test-case-generators/repos.json` from https://github.com/osbuild/osbuild-composer/pull/3288/commits/7684a1bde22fef9321305fe6963996eef05c4f09 to this commit (https://github.com/osbuild/osbuild-composer/pull/3288/commits/21455797fde3c5c39da05a20d8aec759c26314ef) instead.

Reply:
Sure.  Will do.